definitions/mlops-openai_application/definition.yml

```diff
@@ -0,0 +1,20 @@
+domain: MLOPS
+type: OPENAI_APPLICATION
+synthesis:
+  rules:
+    - identifier: applicationName
+      name: applicationName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: OpenAICompletion
+dashboardTemplates:
+  newRelic:
+    template: dashboard.json
+goldenTags:
+  - providerAccountId
```

Comment:
you should define these in the rule tags if you want to create tags based on the attributes.

See: https://github.com/newrelic/entity-definitions/blob/main/docs/synthesis.md#tags

Reply:
Hey @naxhh . Any help here would be appreciated (:

Reply:
@dgoldfine-newrelic I believe that you could define them also under tags on the rules section, instead of defining goldenTags. e.g.

```
synthesis:
  rules:
    - identifier: applicationName
      name: applicationName
      encodeIdentifierInGUID: true
      conditions:
        - attribute: eventType
          value: OpenAICompletion
      tags:
          providerAccountId:
          providerAccountName:
          instrumentation.provider:
```

Examples from the codebase:

1. https://github.com/newrelic/entity-definitions/blob/main/definitions/infra-host/definition.yml#L53
2. https://github.com/newrelic/entity-definitions/blob/main/definitions/infra-container/definition.yml#LL80C1-L80C1

(If you are an internal developer/analyst, you may get a quicker response time asking directly on our #help-entity-platform Slack channel, because we rotate PR reviewers weekly)

Reply:
@otaviocarvalho Thanks for your clarification.
I added `tags` under `rules` like you suggested and removed the `goldenTags` definition. Please let me know if it's ok now.
I'm now working on resolving the dashboard issue and hopefully after that we can merge.